### DISCUSSION TITLE
Együttes képviseleti jog esetén pontosan hogy megy a folyamat?

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#41] [Kérdések és válaszok - Gépi interfész] [answered]
Kedves NAV Szakértők!

azt szeretnénk kérdezni, hogy

- együttes képviseleti jog esetén hogy fog menni a bevallás jóváhagyása?
- hogy lesz arról tudomása az eÁFA M2M rendszernek, ill. az adózó kapcsolódó rendszerének arról, hogy még valakinek hiányzik a jóváhagyása?

Előre is köszönjük!

**Comment by renced42** [comment]
Kedves @szalaierika 

A kettős jóváhagyás nem az eÁfa rendszerben fog megvalósulni. A beadást követően amennyiben az adott bevallás többes jóváhagyással érintett azt majd a NAV ügyfélportálján kell majd "tovább engedni".

**Comment by szalaierika** [comment]
Kedves @renced42 ,

köszi az infót, van esetleg ezzel kapcsolatosan részletes leírás az új dokumentációban? Ha igen, hol találjuk?

Köszi előre is!

**Comment by szalaierika** [comment]
Kedves @renced42 !

Ha jól értjük, akkor a folyamat az alábbi lesz együttes képviselet esetén:

1. az egyik képviselő benyújtja a bevallást az eÁFA M2M rendszerben
2. többi képviseletre jogosult a többes jóváhagyás elvégzése érdekében értesítést kap az eBEV-től, hogy hagyják jóvá ők is
3. ők felmennek a NAV ÜPO-ra és ott jóváhagyják azt
4. (/queryDeclarationProcessingStatus) SUBMITTED statust csak ekkor kaphatja vissza az adózó

Jól értjük, hogy ez a folyamat?

Ha igen, akkor ehhez lenne észrevételünk:
- mivel manuális beavatkozás szükséges itt, ettől kezdve már nem M2M a megoldás
- hogyan lehet ezt a folyamatot lekövetni egy ERP rendszerben?

Véleményünk szerint szükséges lenne kiegészíteni ezt a folyamatot azzal, hogy minden jóváhagyás az ERP rendszerben meg tudjon történni (rendszeren belül), hogy M2M legyen a folyamat.

Köszönjük!

**Comment by renced42** [comment, marked as answer]
Kedves @szalaierika 

Igen a folyamat kb így fog megvalósulni.

Amennyiben a bevallást több személynek is jóvá kell hagynia azt az ÜPO-n lehet majd megtenni. Ennek úgymond koordinálása nem az eÁfa rendszer szkópja, mint ahogyan az ÁNYK kitöltőnek sem az.

Jelen pillanatban a többes jóváhagyást a gépi interfész nem támogatja, de természetesen nem kizárt, hogy ezt később ne lehetne megvalósítani. Ez azonban szakmai döntés elsősorban, nem technikai.

**Reply by szalaierika** [reply]
Köszönöm a megerősítést!
